### PR TITLE
fix: contributors query not pulling all contributors

### LINKED ISSUE #830: [Bug]: contributors query not pulling all contributors
### What happened?

contributors.ts is missing a lot of contributors.

check the contributors script to see why

### What browsers are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by zaCKoZAck0**
## Related Issue

Closes #830

### CODE REVIEW

File: tooling/scripts/contributors.ts [hunk 1]
```diff
@@ -14,13 +14,17 @@ async function getOcto() {
...
 
   const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
-  const response = await octokit.request('GET /repos/{owner}/{repo}/contributors', {
-    owner: 'typehero',
-    repo: 'typehero',
+  
+  // get paginated contributors
+  const res = await octokit.paginate(octokit.rest.repos.listContributors, {
+    owner: "typehero",
+    repo: "typehero",
```
Comment (bautistaaa) [inline tooling/scripts/contributors.ts:21]: is 100 the max

Comment (bautistaaa) [reply tooling/scripts/contributors.ts:21]: if possible it should recursively call if there’s a next page

Comment (zaCKoZAck0) [reply tooling/scripts/contributors.ts:21]: AFAIK It fetches recursively, res will have all 63 (currently) contributors, even if you give page size less than 63, say 20.

Comment (zaCKoZAck0) [reply tooling/scripts/contributors.ts:21]: image

**Review by Hacksore — APPROVED** [review]

**Comment by bautistaaa** [comment]
thanks for fixing fast

**Pull request merged**